src/Resources/scaffolds/6.0/auth.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+use Symfony\Bundle\MakerBundle\FileManager;
+use Symfony\Bundle\MakerBundle\Util\YamlSourceManipulator;
+
+return [
+    'description' => 'Create login form and tests.',
+    'dependents' => [
+        'user',
+        'homepage',
+    ],
+    'packages' => [
+        'profiler' => 'dev',
+    ],
+    'configure' => function(FileManager $files) {
+        $services = new YamlSourceManipulator($files->getFileContents('config/services.yaml'));
+        $data = $services->getData();
+        $data['services']['App\Security\LoginUser']['$authenticator'] ='@security.authenticator.form_login.main';
+        $services->setData($data);
+        $files->dumpFile('config/services.yaml', $services->getContents());
+
+        $security = new YamlSourceManipulator($files->getFileContents('config/packages/security.yaml'));
+        $data = $security->getData();
+        $data['security']['providers'] = [
+            'app_user_provider' => [
+                'entity' => [
+                    'class' => 'App\Entity\User',
+                    'property' => 'email',
+                ],
+            ],
+        ];
```

Comment:
It seems like this change should go into the "user" scaffold, no? We currently add this in `make:user`.